ci/lint/check-dashboard-format.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# Checks front-end code format
+
+set -euxo pipefail
+
+ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE:-$0}")" || exit; pwd)/.."
+WORKSPACE_DIR="${ROOT_DIR}/.."
+
+cd "${WORKSPACE_DIR}"/python/ray/dashboard/client || exit
+
+if [ -z "${BUILDKITE-}" ]; then
```

Comment:
double bracket? `if [[ -z "${BUILDKITE-}" ]]; then`

we can also probably just install npm inside the forge, and remove this.